tests/e2e-instrumentation-test.sh

```diff
@@ -139,33 +133,30 @@ for TEST in $TESTS; do
     ## cleanup at EXIT
 
     found=0
-    cat $SCRIPT_TMP_DIR/build-$$ | grep "Signature ID: $TEST" -B2 | head -3 | grep -q "\*\*\* Detection" && found=1
+    cat $SCRIPT_TMP_DIR/build-$$ | grep $TEST && found=1
```

Comment:
I have not executed this. Is this really "all needed" ? Should we grep for something more complex ? Just worried about an error in stdout with the name of a signature (dunno, cannot parse event XXXX, and we consider it was triggered, or something crazier). Trying to explore what could go wrong, not criticizing.

Reply:
Consider that we're using "everything is an event" single binary here, so the previous string was no longer viable. Not sure if there is a better query I could do. Maybe if there was a stripped down printer only for event names it would be safer.

Reply:
Actually I should just use json output and query the json with eventName. That would likely be best.